packages/slice-machine/core/utils/str.ts

```diff
@@ -1,17 +0,0 @@
-import { createHash } from "crypto";
-
-const camelizeRE = /-(\w)/g;
-
-export function pascalize(str: string): string {
-  if (!str) {
-    return "";
-  }
-  str = str.replace(/_/g, "-").replace(camelizeRE, (c) => {
-    return c ? c.toUpperCase() : "";
-  });
-  return str[0].toUpperCase() + str.slice(1);
-}
-
-export function hash(str: string): string {
```

Comment:
was's some of this used when saving screenshots, and naming slices?

Reply:
I'm not 100% sure where it was being used. I think screenshots were the only place using digests, potentially also for comparing local and remote Slices.

This file was removed in favor of SM UI's `/lib/utils/str.ts` file. However, `hash` did not need to be migrated over.